### PR TITLE
Space Travellers' Hub milestone 8

### PR DESCRIPTION
**Pull request by ambrose-kibet**
## In this milestone we :
- [x]  fetch data from the SpaceX API endpoints
- [x] Render UI: lists
- [x]  Write unit and integration tests for our components

### CODE REVIEW

File: src/pages/Rockets.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getrocket } from '../redux/rocketsSlice';
+import Rocket from '../components/SingleRocket';
+
+const Rockets = () => {
+  const RocketList = useSelector((state) => state.rocket);
```
Comment (vickymarz) [inline src/pages/Rockets.js:7]: - [ ] Your rockets page is currently displaying a blank page signifying that the data is not being fetched from the API. Kindly check it out and ensure you are passing the right data to the API

image

image

image

Comment (vickymarz) [inline src/pages/Rockets.js:7]: - [ ] I think the data is being returned by the API but then it's possible there is an error with the way you are accessing the data from the API. Kindly check carefully.

image

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @ambrose-kibet, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- All tests are passing ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/Rockets.js (hunk 1 above)
Comment (hakichris) [reply src/pages/Rockets.js:7]: thank you @vickymarz , we fixed the problem.

Comment (hakichris) [reply src/pages/Rockets.js:7]: thank you @vickymarz , we fixed the problem.

Comment (Whoistolu) [reply src/pages/Rockets.js:7]: -  Great job team. This has been well resolved.
Screenshot 2023-03-16 at 23 52 21

Screenshot 2023-03-16 at 23 53 57

Screenshot 2023-03-16 at 23 54 19

Comment (Whoistolu) [reply src/pages/Rockets.js:7]: - Well resolved team 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi Team,
@ambrose-kibet and @hakichris 

Great job implementing the recommended changes 👍🏽 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**